src/code-components/ApiProvider/useOnError.ts

```diff
@@ -73,3 +84,12 @@ function handleApiError(
     });
   }
 }
+
+async function getMessage(response: Response): Promise<string | undefined> {
+  let message = undefined;
+  try {
+    const json = await response.json();
+    message = json?.message;
+  } catch {}
```

Comment:
The catch here is useless

Reply:
```suggestion
  } catch (error) {
    throw new ApiError(`response returned unparseable JSON`);
  }
```

Reply:
Maybe I am wrong, but I think it is not useless. 
Even though the catch block is empty, its still necessary to prevent the function from throwing if `response.json()` fails.
Without the `try-catch`, `getMessage` function may throw an error, which would break the rest of the error-handling flow. I just want to ignore it and display toast without description.
In case of a potential error, the toast will not be displayed.

Reply:
I see your point now, let's do this then.
```suggestion
  } catch (error) {
    console.error(`response returned unparseable JSON`, error);
    return undefined;
  }
```